core/ecschema-metadata/src/SchemaFormatsProvider.ts

```diff
@@ -59,7 +60,13 @@ export class SchemaFormatsProvider implements FormatsProvider {
   }
 
   private async getKindOfQuantityFormatFromSchema(itemKey: SchemaItemKey): Promise<SchemaItemFormatProps | undefined> {
-    const kindOfQuantity = await this._context.getSchemaItem(itemKey, KindOfQuantity);
+    let kindOfQuantity: KindOfQuantity | undefined;
+    try {
+      kindOfQuantity = await this._context.getSchemaItem(itemKey, KindOfQuantity);
```

Comment:
getSchemaItem will return undefined. What errors are we catching here?

Reply:
When the formatsProvider uses a SchemaContext that has an ECSchemaRpcLocater registered, the rpc requests for getting a schema/schemaItem will throw if the item isn't found. A new workflow for retrieving a schemaContext is through the [IModelConnection](https://github.com/iTwin/itwinjs-core/blob/675bd9e1e323f1281ba8133722b2e7fd83be9f4f/core/frontend/src/IModelConnection.ts#L630-L636), so this error will popup and be thrown through `RpcRequest`